src/components/Header/LayerRoute/LayerRoute.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+
+import useMap from '@hook/useMap';
+import { useSelectedState } from '@store/state/selectedContext';
+import { useSelectedDistrict } from '@store/district/selectedContext';
+
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+
+import * as Styles from './styles';
+
+const LayerRoute = () => {
+  const [selectedLayer, setSelectedLayer] = useState('country');
+
+  const { resetMapValues, resetDistrictValues } = useMap();
+  const { selected: selectedState } = useSelectedState();
+  const { selected } = useSelectedDistrict();
+
+  useEffect(() => {
+    if (selected) {
+      setSelectedLayer('district');
+    } else if (selectedState && !selected) {
+      setSelectedLayer('state');
+    } else {
+      setSelectedLayer('country');
+    }
+  }, [selected, selectedState]);
+
+  const returnPath = () => {
+    if (selectedLayer === 'district') {
+      return (
+        <>
+          &nbsp;-&nbsp;
+          <div className="place state" onClick={() => resetDistrictValues()}>
```

Comment:
Did you try to create a button from mui?

Reply:
oops, forgot about that component 😅